multimodal/image_question_answering/.env.example

```
@@ -0,0 +1 @@
+﻿GOOGLE_API_KEY=your_api_key_here
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)